.gitignore

```diff
@@ -7,6 +7,9 @@
 # Don't ignore the LICENSE file
 !LICENSE
 
+# Don't ignore the README.md file
```

Comment:
Given we are adding things to the .gitignore - I presume there are files that we really do want to ignore? Just wondering if we should just not ignore everything, or if we might be better approaching from the other direction - add ingnores for the things we really do want to ignore?